### PR TITLE
Adds the capability to build and deploy containers

### PR DESCRIPTION
**Pull request by pboothe**
No cloud auth (yet) for travis, but it works locally which seems like a good time to checkpoint things.

### CODE REVIEW

File: build_and_deploy_container.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
```
Comment (stephen-soltesz) [inline build_and_deploy_container.sh]: Please add a one or two sentence docstring here.

File: build_and_deploy_container.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+set -e
+set -x
+
+IMAGE_TAG=${1:?Please provide a tag for the image}
```
Comment (stephen-soltesz) [inline build_and_deploy_container.sh]: I suggest using the `USAGE=` trick like the other deploy scripts for giving a hint about the other parameters if we forget the first.

**Comment by stephen-soltesz** [comment]
Cool. LGTM. Just some suggestions for helpful comments.

File: build_and_deploy_container.sh (hunk 1 above)
Comment (pboothe) [inline build_and_deploy_container.sh]: Done

**Pull request merged**